src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -239,8 +240,12 @@ public boolean hasNoOngoingBusinessProcess() {
     private final LocalDateTime claimDismissedDate;
     private final String claimAmountBreakupSummaryObject;
     private final LocalDateTime respondent1LitigationFriendDate;
-    private final LocalDateTime respondent1LitigationFriendCreatedDate;
+    private final LocalDateTime respondent2LitigationFriendDate;
+    private final LocalDateTime genericLitigationFriendDate;
```

Comment:
This field should never be required - as we only set Dates for Respondent 1 and 2